ERCS/erc-7540.md

```diff
@@ -330,6 +384,30 @@ MUST be emitted when a redemption Request is submitted using the `requestRedeem`
       type: uint256
 ```
 
+#### `OperatorSet`
+
+The `owner` has set the `approved` status to an `operator`.
+
+MUST be logged when the operator status is set.
+
+MAY be logged when the operator status is set to the same status it was before the current call.
+
+```yaml
+- name: OperatorSet
+  type: event
+
+  inputs:
+    - name: owner
+      indexed: true
+      type: address
+    - name: operator
```

Comment:
Why not keep the naming consistent with ERC6909 and change operator to spender

Reply:
Spender seemed wrong to me here since it can do more than transfer tokens of the owner, it's more generic.

Also worth noting that solmate's implementation actually doesn't use spender: https://github.com/transmissions11/solmate/blob/main/src/tokens/ERC6909.sol#L77 So this seemed like a good alternative to me

Reply:
But I'm happy to change to spender if you think that's better

Reply:
seems fine as is